packages/adp-tooling/jest.setup.js

```diff
@@ -0,0 +1,13 @@
+// jest.setup.js
+
+// Mocking the '@zowe/secrets-for-zowe-sdk' module to simulate keyring functionality
+// This ensures that tests for the KeyStoreManager class which utilizes '@zowe/secrets-for-zowe-sdk'
+// does not have any side effects while testing. 
+jest.mock('@zowe/secrets-for-zowe-sdk', () => ({
+    keyring: {
+        setPassword: jest.fn(),
+        getPassword: jest.fn(),
+        deletePassword: jest.fn(),
+        findCredentials: jest.fn(),
+    }
+}));
```

Comment:
does setting `process.env.FIORI_TOOLS_DISABLE_SECURE_STORE` also work?

Reply:
Setting `process.env.FIORI_TOOLS_DISABLE_SECURE_STORE` to any value will force the use of the DummyStore, which is useful for tests...

Reply:
Thanks for pointing this out .. I use `process.env.FIORI_TOOLS_DISABLE_SECURE_STORE` in adpTooling now.